dts/x86/quark_d2000_crb.dts

```diff
@@ -0,0 +1,29 @@
+/dts-v1/;
+
+#include "intel_quark_d2000.dtsi"
+
+/ {
+	model = "Mintvalley board";
```

Comment:
seems like mintvalley is an internal name for the board, not seeing that referenced in our board docs for the board.  Probably not the best choice for model.